src/Adapter/Currency/CommandHandler/AddCurrencyHandler.php

```diff
@@ -26,13 +26,17 @@
 
 namespace PrestaShop\PrestaShop\Adapter\Currency\CommandHandler;
 
+use Context;
 use Currency;
+use Language;
+use PrestaShop\PrestaShop\Adapter\SymfonyContainer;
```

Comment:
Unneeded (see below)

Reply:
Unneeded because unused you mean? Shouldn't the CSFixer remove this?

Reply:
it should.
it didn't.
I do.

Reply:
That's weird, so the CSFixer has issues? Or you just forgot it?

Reply:
I forgot it, but it used to bloc travis test, wasn't it ?

Reply:
Yes but now the lint is out of the travis tests, and performed by github actions
But we need to check our workflow as the checks are not performed on PRs nor do they block the merge..